.github/workflows/ci.yml

```diff
@@ -24,4 +24,4 @@ jobs:
     - uses: actions/checkout@master
     - uses: gaurav-nelson/github-action-markdown-link-check@v1
       with:
-          config-file: '.github/ci/markdown-link-check.json'
+          config-file: '.github/workflows/markdown-link-check.json'
```

Comment:
This is a surprising fix...

Reply:
@rsandifo-arm - I think it would make sense to submit this as a generic bugfix of the CI configuration in a separate PR.